app/controllers/posts_controller.rb

```diff
@@ -4,5 +4,9 @@ def index
     @posts = @user.recent_3_posts
   end
 
-  def show; end
+  def show
+    @user = User.find(params[:user_id])
+    @post = @user.posts.find(params[:id])
+    @comments = @post.comments.order('created_at')
+  end
```

Comment:
- [ ] Create current_user method on `application_controller` 
```ruby
# app/controllers/application_controller.rb
...

def current_user
  User.first
end
```

Reply:
- [ ] Please create the new and create actions on the post_controller

Reply:
- [ ] In your views please add the respective form in the views/posts/new.html.erb file which accepts input for creating posts.